src/components/UI/Logo.js

```diff
@@ -0,0 +1,200 @@
+          letter-spacing="0"
+        ></path>
+      </svg>
+    );
+  };
+
+  return {
+    getLogo,
```

Comment:
What is the difference between getLogo and getAgricolaLogo?

Reply:
Getlogo Is without text "agricola"